app/src/main/java/org/mozilla/fenix/library/history/HistoryView.kt

```diff
@@ -131,6 +135,13 @@ class HistoryView(
             )
             recentlyClosedNav.isVisible = !userHasHistory
         }
+        with(binding.syncedHistoryNavEmpty) {
+            val syncedHistoryVisible = FeatureFlags.syncedHistoryFeature && !userHasHistory
```

Comment:
```suggestion
            val syncedHistoryVisible = FeatureFlags.syncedHistoryFeature && userHasHistory
```
Shouldn't we want users to have history?

Reply:
It works a bit differently, see the way `recentlyClosedNav.isVisible = !userHasHistory` works just right above ?
There are the elements that are part of the view itself and are displayed when the view is in empty state.
The elements that are inside the list are actually part of a viewholder.